mscore/scoreBrowser.cpp

```diff
@@ -298,5 +306,179 @@ void ScoreBrowser::setScoreActivated(QListWidgetItem* val)
       emit scoreActivated(item->info().filePath());
       }
 
+//---------------------------------------------------------
+//   focusNextPrevChild
+//---------------------------------------------------------
+
+bool ScoreBrowser::focusNextPrevChild(bool next)
+      {
+      Q_UNUSED(next);
+      return false; // disable tab action during template navigation
+      }
+
+void ScoreListWidget::keyPressEvent(QKeyEvent* event)
+      {
+      int idx = mscore->getNewWizard()->getTemplateFileBrowser()->getScoreLists().indexOf(this);
```

Comment:
the scoreListwidget shouldn't know about the newscorewizard.

Reply:
I wrote this because the last template was getting selected on using Shift+Tab, though I had disabled the default tab behavior. Also, the keyPressEvent was being sent to ScoreListWidget instead of ScoreBrowser. I'm forwarding the event to ScoreBrowser::keyPressEvent.

Reply:
OK, but there's got to be another way of solving that problem.  Like if you can't get there by finding the parent directly, maybe we could add a parent field to the ScoreListWidget?

Also in the code right below this, it looks like you are hard-coding "7" as the number of rows, but that isn't safe either - as for the code dealing with key signature, time signature, and symbol sections of the master palette, it needs to be handled in a way that doesn't depend on the current arrangements.

Reply:
Okay, I will try to find another solution.